platform/plugins/pid/index.ts

```diff
@@ -1,5 +1,4 @@
-import { KibanaPlugin } from '../../server/plugins/types';
-import { KibanaPluginFeatures } from '../../types';
+import { KibanaPlugin, KibanaPluginFeatures } from '../../server/plugins/types';
 import { Logger } from '../../logger';
 
 import { PidConfig } from './PidConfig';
```

Comment:
Definition of the class without name (below in this file) makes my IDE mad :) Do they (TS Team) intentionally support such a construct?

![screenshot from 2017-07-20 20-28-17](https://user-images.githubusercontent.com/1713708/28432951-3dc76622-6d8a-11e7-8ba1-aad3a5dc7603.png)